container.go

```diff
@@ -158,6 +158,13 @@ func (c *Container) Remove(rem CanvasObject) {
 	}
 }
 
+// RemoveAll updates the contents of this container to no longer include any objects.
+//
+// Since: 2.2
+func (c *Container) RemoveAll() {
+	c.Objects = nil
```

Comment:
I think it should Refresh() as well, method calls normally make sure we’re updating after the change.